server/pages/index.js

```diff
@@ -119,7 +122,19 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getFileContentsSafely( filepath ) {
+	return new Promise( function( resolve ) {
+		fs.readFile( filepath, function( err, content ) {
```

Comment:
@DanReyLop This is actually the server-side rendering code. It executes at runtime, not build time.

I do like the simplicity of `fs.readFileSync`. But given that this runs on every page load, I feel like async IO is The Right Thing to do.

What do you think?